### PR TITLE
USHIFT-1387: Execute metal test step remotely in the hypervisor

### PR DESCRIPTION
**Pull request by pacevedom**

### CODE REVIEW

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests

File: ci-operator/step-registry/openshift/microshift/e2e/metal-tests/openshift-microshift-e2e-metal-tests-commands.sh [hunk 1]
```diff
@@ -168,12 +168,16 @@ EOF
...
 trap 'scp -r ${INSTANCE_PREFIX}:/home/${HOST_USER}/microshift/_output/test-images/scenario-info ${ARTIFACT_DIR}' EXIT
 
-# Run the scenario tests, if the phase script exists
-# (we can clean this up after the main PR lands)
-cd /microshift/test || true
+cat > /tmp/tests.sh << EOF
+cd ~/microshift/test
 if [ -f ./bin/ci_phase_test.sh ]; then
-    ./bin/ci_phase_test.sh
+  ./bin/ci_phase_test.sh
```
Comment (pmtk) [inline ci-operator/step-registry/openshift/microshift/e2e/metal-tests/openshift-microshift-e2e-metal-tests-commands.sh:174]: Do you think we could remove the `if [ -f ./bin/ci_phase_test.sh ]; then`? Maybe in another PR?
The file is in microshift's main, so it's not needed

**Comment by pmtk** [comment]
/lgtm
/hold

unhold at your convenience

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests

**Pull request merged**